health/micro-ui-react19/web/packages/css/src/pages/employee/attendance.scss

```
@@ -4,6 +4,11 @@
   white-space: nowrap;
   font-weight: 700;
   line-height: normal;
+
+  &.register-details{
+    font-size: var(--digitv2-fontSize-heading-m-desktop);
+    font-family: 'Roboto';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)